templates/reporting/report_user_list.html.twig

```diff
@@ -22,6 +32,9 @@
             {% else %}
                 {% set absoluteTotals = 0 %}
                 {% set totals = {} %}
+                {% set fieldName = form.sumType.vars.value %}
+                {% set totalFieldName = 'total' ~ (fieldName | capitalize) %}
```

Comment:
Use of capizalize formatter doesn't feel right here, but gets the job done.... is this acceptable?